src/actions/Stake.actions.ts

```diff
@@ -61,16 +87,22 @@ export const changeApproval =
   };
 
 export const changeStake =
-  ({ action, value, provider, address, networkID }) =>
-  async dispatch => {
+  ({ action, value, provider, address, networkID }: IChangeStake) =>
+  async (dispatch: Dispatch) => {
     if (!provider) {
       alert("Please connect your wallet!");
       return;
     }
 
     const signer = provider.getSigner();
-    const staking = await new ethers.Contract(addresses[networkID].STAKING_ADDRESS, OlympusStaking, signer);
-    const stakingHelper = await new ethers.Contract(addresses[networkID].STAKING_HELPER_ADDRESS, StakingHelper, signer);
+
+    // TS-REFACTOR-NOTE: types arg accepted mismatch
```

Comment:
Whats the mismatch? Is that bad?

Reply:
I believe the mismatch was w/ the `unstake` and `stake` functions, the args we've provided vs. expected are inconsistent. I wouldn't say it's bad because staking/unstaking clearly work, but it is strange.